src/js.rs

```diff
@@ -172,6 +172,13 @@ extern "C" {
     #[wasm_bindgen(method)]
     pub fn find(this: &Array, predicate: &mut FnMut(JsValue, u32, Array) -> bool) -> JsValue;
 
+    /// The findIndex() method returns the index of the first element in the array that 
+    /// satisfies the provided testing function. Otherwise -1 is returned.
+    /// 
+    /// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/findIndex
+    #[wasm_bindgen(method, js_name = findIndex)]
+    pub fn find_index(this: &Array, predicate: &mut FnMut(JsValue, u32, Array) -> bool, this_arg: &JsValue) -> u32;
```

Comment:
I don't think we need `this_arg` because our closure can't access it. Want to do a follow up to remove it?

Reply:
Thank you for your Advice.
I'll try to remove it 😃